package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "movie-app",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-dev-server --mode development --open --hot",
+    "prod": "webpack --mode production"
+  },
+  "dependencies": {
+    "@babel/preset-env": "^7.16.8",
+    "@babel/preset-react": "^7.16.7",
```

Comment:
These packages should be in `devDependencies`

Reply:
fixed